desktop/libs/aws/src/aws/conf.py

```diff
@@ -349,12 +346,14 @@ def has_iam_metadata():
 
 def has_s3_access(user):
   from desktop.auth.backend import is_admin
+
+  return user.is_authenticated and user.is_active and (is_admin(user) or user.has_hue_permission(action="s3_access", app="filebrowser"))
+
```

Comment:
This will break now, we need to return True on a RAZ cluster..

Reply:
added the raz.is_enabled back to the old state.